libs/designer/src/lib/core/utils/connectors/connections.ts

```diff
@@ -83,7 +83,9 @@ export function getAssistedConnectionProps(connector: Connector, manifest?: Oper
       description: 'Label for function app selection',
     });
 
-    const getColumns = (functionApp: any) => [functionApp?.name, functionApp?.properties?.resourceGroup, functionApp?.location];
+    const getColumns = (functionApp: any) => [getObjectName(functionApp), functionApp?.properties?.resourceGroup, functionApp?.location];
+
+    const getSubResourceName = (azureFunction: any) => getObjectName(azureFunction).split('/').slice(1).join('/');
```

Comment:
can you give an example here?
are we just getting the function name from function app? 

And also we should be handling for null refs here

Reply:
This can be reverted, it was for back when I was showing both swagger functions in this component, the swagger names had a different format

Reply:
Update on this, the function name value is `{appName}/{functionName}`, the extra split and slice was just to remove the app name since some are pretty long and it's already going to be known since they had to click on it